src/main/java/com/github/algobot76/surabaya/util/Parameter.java

```diff
@@ -9,4 +9,9 @@
 
 	private final String type;
 
+	public Parameter(String name, String type) {
+		this.name = name;
+		this.type = type;
+	}
+
```

Comment:
There is no need to specify the constructor because this will be automatically generated by Lombok.